### PR TITLE
[LMS-335][FE]Course Detail Header

### PR DESCRIPTION
**Pull request by epifaniovillaro**
## Issue Link
[LMS-335](https://framgiaph.backlog.com/view/LMS-335)

## Defintion of Done

- Able to create the layout according to Figma design 
- Accepts dynamic content
- Able to display the three dots (not functional)

## Steps to reproduce
http://localhost:3000/trainer/course/detail/[id]

## Affected Components / Functionalities / Page
trainer/course/detail/[id].tsx

## Test Cases
_will update soon..._

## Notes
n/a

## Screenshots
image
image

### CODE REVIEW

File: client/src/pages/trainer/course/detail/[id].tsx [hunk 1]
```diff
@@ -1,15 +1,102 @@
...
+import Navbar from '@/src/shared/components/Navbar';
+import Button from '@/src/shared/components/Button';
+import { dropdownItems, navItems } from '@/src/pages/demo/layouts/navbar';
 
 const CourseDetails: React.FC = () => {
-  const router = useRouter();
-  const { id } = router.query;
+  const tabData = [
+    { id: 1, tabName: 'Course Content' },
+    { id: 2, tabName: 'Knowledge Boosters' },
```
Comment (ragg09-sunasterisk) [inline client/src/pages/trainer/course/detail/[id].tsx:11]: We dont need "knowledfe Booster" and "Course Layout", Please remove this

Comment (ragg09-sunasterisk) [inline client/src/pages/trainer/course/detail/[id].tsx:11]: we dont need "knowledge booster" and "Course Layout", please remove this

**Review by Marielleregner7 — APPROVED** [review]
**APPROVED**
- [ ] Course Header is visible and is identical to the Wireframe.
- [ ] Tab is working, and if a tab is clicked it displays the tab's content.
- [ ] No ESLint warnings or errors.
- [ ] No error found, when running npm run build.

Comments: 1
Test time: 35 mins
Iterations: 1

Proof:
courseheaderpif
image

**Review by ragg09-sunasterisk — APPROVED** [review]

**Comment by ragg09-sunasterisk** [comment]
Comments: 1
Iterations: 1

**Pull request merged**